ddapm_test_agent/agent.py

```diff
@@ -898,13 +1032,15 @@ def make_app(
             web.post("/test/session/responses/config", agent.handle_v07_remoteconfig_create),
             web.post("/test/session/responses/config/path", agent.handle_v07_remoteconfig_path_create),
             web.put("/test/session/responses/config", agent.handle_v07_remoteconfig_put),
+            web.put("/test/session/integrations", agent.handle_put_tested_integrations),
```

Comment:
these endpoints need to be documented in the README, especially with the saving-seen-integrations-to-file functionality that exists that writes the integrations to some magic file.

Reply:
also nice to have some tests

Reply:
also a release note